src/library_pthread.js

```diff
@@ -992,9 +992,8 @@ var LibraryPThread = {
     else PThread.threadExit(status);
     // pthread_exit is marked noReturn, so we must not return from it.
     if (ENVIRONMENT_IS_NODE) {
-      // exit the pthread properly on node, as a normal JS exception will halt
-      // the entire application.
-      process.exit(status);
+      // set the exit code while letting the process exit gracefully
+      process.exitCode = status;
```

Comment:
What does this do exactly?

pthread_exit should not effect he whole process, right?  Only the current thread

Reply:
Good point, removed with commit https://github.com/kleisauke/emscripten/commit/2831f73927d9240ddf94e1127f1f8d5464ff859e.